pkg/executor/util/util.go

```diff
@@ -152,3 +157,39 @@ func GetObjectReaperInterval(logger *zap.Logger, executorType fv1.ExecutorType,
 func getExecutorEnvVarName(executor fv1.ExecutorType) string {
 	return strings.ToUpper(string(executor)) + "_OBJECT_REAPER_INTERVAL"
 }
+
+func CreateDirAndFile(fileName string, logger *zap.Logger) (*os.File, error) {
```

Comment:
Just keep this function simple for now.


```
func CreateDumpFile(logger *zap.Logger) (*os.File, error)
```

Function should simply generate filename and create file at that path.